### DISCUSSION TITLE
Christmas movies

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#309] [General]
Which one is your favorite? Did you hear of some new good Christmas movies maybe?

**Comment by ellielo** [comment]
Miracle on 34th Street is the best Christmas movie of all the times. If you have a firestick you not gonna have troubles finding it. In case you need to "clean up" your firestick cause you have too many addons check this guide [https://www.firesticktricks.com/delete-apps-from-firestick.html](url). It explains everything step by step so even not the smartest from us can do it :)

**Comment by Ozankey** [comment]
The Nightmare Before Christmas.  Amazing animation and very fun story.
